File: servers/bar.php
<?php
class Bar {
  var $val1;
  var $val2;
  function __construct($arg1, $arg2) {
    $this->val1 = $arg1;
    $this->val2 = $arg2;
  }
  
  function find($arg) { return array('1', '2'); }
  function locate() { return ''; }
}
?>

File: servers/bar.rb
class Bar
  attr_accessor :val1, :val2
  attr_accessor :foo, :baz

  def initialize(v1, v2)
    @val1 = v1
    @val2 = v2
  end
  def baz?
    false
  end
  def self.find(count)
    count
  end
  def self.locate
    'y'
  end
end